letsencrypt-auto-source/tests/oraclelinux6_tests.sh

```diff
@@ -27,55 +24,31 @@ if [ $RESULT -ne 0 ]; then
   exit 1
 fi
 
-echo "PASSED: bootstrap from old letsencrypt-auto succeeded and installed Python 3.4"
+echo "PASSED: Bootstrap from old letsencrypt-auto succeeded and installed Python 3.4"
 
-# Expect letsencrypt-auto to just fail to rebootstrap and start certbot in interactive
-# shell since SCL is not installed.
-if ! "$LE_AUTO" --version 2>&1 | grep -q "Enable the SCL repository and try running Certbot again."; then
+# Expect letsencrypt-auto to skip rebootstrapping with a warning since SCL is not installed.
+if ! "$LE_AUTO" --non-interactive --version 2>&1 | grep -q "Skipping upgrade because new OS dependencies may need to be installed."; then
   echo "FAILED: Script letsencrypt-auto managed to start Certbot in interactive shell while SCL is not enabled!"
   exit 1
 fi
 
-echo "PASSED: Script letsencrypt-auto did not rebootstrap and did not star Certbot."
-
-# Expect letsencrypt-auto to not update certbot, but start it anyway in non-interactive
-# shell since SCL is not installed.
-# NB: Readline has an issue on all Python versions for CentOS 6, making `certbot --version`
-# output an unprintable ASCII character on a new line at the end.
-# So we take the second last line of the output.
-version=$($LE_AUTO --version 2>/dev/null | tail -2 | head -1)
-
-if [ -z "$version" ]; then
-  echo "ERROR: Script letsencrypt-auto failed to start certbot in a non-interactive shell while SCL was not enabled."
-  exit 1
-fi
-
-if [ "$version" != "$INITIAL_CERTBOT_VERSION_PY34" ]; then
-  echo "ERROR: Script letsencrypt-auto upgraded certbot in a non-interactive shell while SCL was not enabled."
-  exit 1
-fi
-
-echo "PASSED: Script letsencrypt-auto did not upgraded certbot but started it successfully while SCL was not enabled."
+echo "PASSED: Script letsencrypt-auto did not rebootstrap."
 
 # Enable SCL
 yum install -y oracle-softwarecollection-release-el6 >/dev/null
 
-# Following test is exectued in a subshell, to not leak any environment variable
-(
-  export VENV_PATH=$(mktemp -d)
 
-  # Expect letsencrypt-auto to bootstrap successfully since SCL is available
-  "$LE_AUTO" -n >/dev/null 2>/dev/null
+# Expect letsencrypt-auto to bootstrap successfully since SCL is available.
+"$LE_AUTO" -n --version >/dev/null 2>/dev/null
```

Comment:
This is in response to https://github.com/certbot/certbot/pull/7268#discussion_r335752153.

Ah you're right. I think the reason it works in our test farm is we're making sure we don't print any output when there's no bootstrapping/upgrading to do. Can we add a similar test here by calling `$LE_AUTO` again after this since we won't be testing on RHEL/OL/SL 6 in our test farm?

Reply:
This is not work as it. Even with `--quiet` flag, the script will echo a warning about the insecure permissions. Should I:
* consider that as a bug, and fix it in this PR
* consider that as a bug, and let another PR fix that, and not add the test here
* consider that as a feature, and set correctly the permissions in the test to not have the warning?